Cargo.toml

```
@@ -28,10 +28,7 @@ libc = "0.2"
 log = "0.4.4"
 pathfinder_geometry = "0.5"
 pathfinder_simd = "0.5.5"
-
-[dependencies.freetype]
-version = "0.7"
-optional = true
+freetype-sys = {version = "0.23", optional = true}
```

```suggestion
freetype-sys = { version = "0.23", optional = true }
```

Also this should be moved below `float-ord = "0.3"`.